PluginManager/po/it-local.po

```diff
@@ -0,0 +1,265 @@
+# Italian translation for Gramps PluginManager addon
+#
+# This file is distributed under the same license as the Gramps package.
+#
+# Giansalvo Gusinu <giansalvo.gusinu@gmail.com>, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-01-22 09:28-0600\n"
+"PO-Revision-Date: 2019-02-10 03:00+0100\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Last-Translator: Giansalvo Gusinu\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.2.1\n"
+
+#: PluginManager/PluginManager.gpr.py:29
+msgid "Plugin Manager Enhanced"
+msgstr "Gestore Plugin Avanzato"
+
+#: PluginManager/PluginManager.gpr.py:30
+msgid "An Addon/Plugin Manager with several additional capabilities"
+msgstr "Un Gestore di Addon/Plugin con varie funzionalità addizionali"
+
+#: PluginManager/PluginManager.py:87
+msgid "Plugin Manager - Enhanced"
+msgstr "Gestore dei Plugin - Avanzato"
```

Comment:
Gestore dei plugin - Avanzato
(but I would go with lowercase as well for avanzato)